### PR TITLE
Sign up user

### PR DESCRIPTION
**Pull request by SeanEmmers**
https://trello.com/c/VpDW5aKC/9-xxxl-users-can-sign-up

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -35,3 +35,60 @@
...
+      font-size: 1.4em;
+      text-align: left;
+      letter-spacing: -1px;
+      margin-bottom: 3px;
+      margin-top: 0px;
+    }
+  }
+}
+
+.gravatar {
```
Comment (merrynhr) [inline app/assets/stylesheets/application.scss:67]: Really nice work

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module UsersHelper
+
+  # Returns the Gravatar for the given user.
+  def gravatar_for(user)
+    p gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
```
Comment (merrynhr) [inline app/helpers/users_helper.rb]: Do we still need ps?

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Welcome User :) </h1>
+<p>Find me in app/views/users/index.html.erb</p>
```
Comment (merrynhr) [inline app/views/users/index.html.erb]: Should get rid of this line now?

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, @user.name) %>
```
Comment (merrynhr) [inline app/views/users/show.html.erb:1]: Interesting, will look into this more!

File: config/routes.rb [hunk 5]
```diff
@@ -4,4 +4,8 @@
   resources :posts do
     get '/like', to: 'posts#like', on: :member
   end
+
+  resources :users do
```
Comment (merrynhr) [inline config/routes.rb:8]: Do we need this comment?

**Review by merrynhr — CHANGES_REQUESTED** [review]
Amazing work guys! Really impressive :)

File: app/helpers/users_helper.rb (hunk 2 above)
Comment (YinnyF) [reply app/helpers/users_helper.rb]: no we don't, I have now removed

File: config/routes.rb (hunk 5 above)
Comment (YinnyF) [reply config/routes.rb:8]: removed thanks

File: app/views/users/index.html.erb (hunk 3 above)
Comment (YinnyF) [reply app/views/users/index.html.erb]: removed the file

**Comment by merrynhr** [comment]
Looking good!

**Pull request merged**